scripts/buildwithfdpp.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+# Notes:
+#    1/ Copy this script to the top directory of the dosemu source,
+#       as it won't exist in earlier versions and will be removed
+#       during bisect checkouts.
+#    2/ Change the above variable to match your path to FDPP git location.
+#    3/ This script has been tested with dosemu2 / fdpp combinations to
+#       API 26..13 tags okay to configure, build and run FDPP plugin.
+#       API 12..11 tags okay to configure, build and run but with FS issues
+#       API  <= 10 tags not tested
+#
+#    Usage: ./buildwithfdpp.sh /path/to/fdpp.git --matchapi
+
+if [ ! -d src ] ; then
+  echo Run this script from the top directory of the dosemu2 source
+  exit 3
+fi
+
+FDPP_LIBRARY_SOURCE=$(realpath "$1")
+if [ ! -d ${FDPP_LIBRARY_SOURCE} ] ; then
+  echo Local FDPP git directory \"${FDPP_LIBRARY_SOURCE}\" does\'t exist
+  exit 2
+fi
+
+FDPP_PLUGIN_SOURCE=src/plugin/fdpp/fdpp.c
+if [ ! -f ${FDPP_PLUGIN_SOURCE} ] ; then
+  echo FDPP plugin was not implemented at this version of Dosemu2
+  exit 1
+fi
+
+FDPP_PLUGIN_VER=$(awk -F= '/FDPP_API_VER !=/{printf("%d", $2);}' ${FDPP_PLUGIN_SOURCE})
+FDPP_LIBRARY_VER=$(awk -F" " '/#define FDPP_API_VER/{printf("%d", $3);}' "${FDPP_LIBRARY_SOURCE}/fdpp/thunks.h")
+
+if [ ! "${FDPP_PLUGIN_VER}" = "${FDPP_LIBRARY_VER}" ] ; then
+  if [ ! "$2" = "--matchapi" ] ; then
+    echo FDPP version mismatch in "${FDPP_LIBRARY_SOURCE}"
+    echo need API"${FDPP_PLUGIN_VER}" but have API"${FDPP_LIBRARY_VER}"
+    exit 1
+  fi
+  echo Building FDPP at API${FDPP_PLUGIN_VER}
+  (cd ${FDPP_LIBRARY_SOURCE} && git checkout -f API${FDPP_PLUGIN_VER} && make clean all)
```

Comment:
local_install_hdrs should be here.

Reply:
That target doesn't exist on older FDPP, or does it?

Reply:
Already does. :)
Sorry, very bad internet connection
here these days. Don't see all comments.

Reply:
I see that target added at commit f8615a1cb, but that's not older FDPP. The aim of the script is to be able to build from current API back to 11ish. For instance if I try to build API18 I get this:
~~~
make[1]: Leaving directory '/clients/common/fdpp.git/fdpp'
make: *** No rule to make target 'local_install_hdrs'. Stop.
~~~
If you want to make it a prerequisite of the all target, that would work.

Reply:
Ah, cool then.
Please squash and lets apply.

Reply:
squashed now